cmd/zetaclientd/start.go

```diff
@@ -341,14 +349,18 @@ func start(_ *cobra.Command, _ []string) error {
 	//	defer zetaSupplyChecker.Stop()
 	//}
 
-	startLogger.Info().Msgf("awaiting the os.Interrupt, syscall.SIGTERM signals...")
-	ch := make(chan os.Signal, 1)
-	signal.Notify(ch, syscall.SIGINT, syscall.SIGTERM)
-	sig := <-ch
-	startLogger.Info().Msgf("stop signal received: %s", sig)
+	startLogger.Info().Msgf("zetaclientd is running")
 
-	zetacoreClient.Stop()
+	<-notifyCtx.Done()
 
+	startLogger.Info().Msgf("initiating zetaclientd shut down")
+
+	//stop chain observers
+	for _, observer := range observerMap {
+		observer.Stop()
+	}
+	zetacoreClient.Stop()
```

Comment:
Latest `develop` doesn't have observer.Stop(), only `zetacoreClient.Stop()` https://github.com/zeta-chain/node/blob/develop/cmd/zetaclientd/start.go#L350. Please make sure this is up-to date with develop